src/util/numeric.h

```diff
@@ -149,6 +149,16 @@ inline void sortBoxVerticies(v3s16 &p1, v3s16 &p2) {
 		SWAP(s16, p1.Z, p2.Z);
 }
 
+inline void sortBoxVerticies(v3f &p1, v3f &p2)
```

Comment:
It's just an overload -- it does the exact same thing just with a different type.  This is what overloads are for, we don't need different names for different types like in C.